src/main/java/cn/nukkit/entity/EntityLiving.java

```diff
@@ -208,7 +208,7 @@ public boolean entityBaseTick(int tickDiff) {
             }
 
             if (!this.hasEffect(Effect.WATER_BREATHING) && this.isInsideOfWater()) {
```

Comment:
actually you need add here, not on first if, but k

Reply:
There's no difference between Player#gamemode and Player#getGamemode. A getter stays a getter.